ui/push-health/TestFailure.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faStar } from '@fortawesome/free-solid-svg-icons';
+import { Badge } from 'reactstrap';
+
+import { getJobsUrl } from '../helpers/url';
+
+export default class TestFailure extends React.PureComponent {
+  render() {
+    const { failure, repo, revision } = this.props;
+    const {
+      testName,
+      jobName,
+      jobId,
+      classification,
+      failureLine,
+      confidence,
+    } = failure;
+
+    return (
+      <div className="d-flex flex-column mt-2 mb-3 ml-2">
```

Comment:
Could you switch this to reactstrap instead?

Reply:
Yeah, good call.  Done.